_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -40,8 +40,9 @@
 # for the collection exercise and don't represent event data
 ignore_columns = ['surveyRef', 'exerciseRef']
 CSRF_REGEX = re.compile(r'<input id="csrf_token" name="csrf_token" type="hidden" value="(.+?)"\/?>')
-USER_WAIT_TIME_MIN_SECONDS = 5
-USER_WAIT_TIME_MAX_SECONDS = 15
+# USER_WAIT_TIME_WAIT_TIME is between GET and POST requests
+USER_WAIT_TIME_MIN_SECONDS = int(os.getenv('user_wait_time_min_seconds'))
+USER_WAIT_TIME_MAX_SECONDS = int(os.getenv('user_wait_time_max_seconds'))
```

Comment:
Can these default to 5 and 15 if the env var isn't provided?

Reply:
Updated the config